src/ebml/macros.rs

```diff
@@ -0,0 +1,68 @@
+#[macro_export]
```

Comment:
Do we want to make these macros public? If not, we can remove this attribute for all of them

Reply:
Good point. I guess I'll use macro_use instead. Macro scoping confuses me sometimes...

Reply:
IIRC you can import macros as functions in the necessary module or file

Reply:
Yeah, I found a way to do it. Macros are not very intuitive to use...